docs/pages/docs/core-concepts/filtering.mdx

```diff
@@ -8,55 +8,67 @@ import HeartIcon from "@heroicons/react/solid/HeartIcon";
 
 # Filtering Workspaces
 
-Turborepo supports a `pnpm`-like `--filter` flag that allows you to select the workspaces
-that will act as "entry points" into your monorepo's workspace/task graph.
-You can filter your project by workspace name, workspace directory, whether workspaces include dependents/dependencies, and by changes
-in git history.
+A monorepo can contain hundreds, or thousands, of workspaces. By default, running `turbo run test` will execute the `test` task in **all available workspaces**.
 
-`turbo` will run each task against each matched workspace, ensuring that any dependencies
-are run first, according to the `pipeline` specification in [`turbo.json`](/docs/reference/configuration#pipeline).
+![Without using a filter, test will be run across all packages](../../../public/images/docs/no-filter.png)
 
-## Filter Syntax
+Turborepo supports a `--filter` flag that lets you **select the workspaces you'd like to execute your task in**.
+
+![With a filter on shared, only the shared package runs test](../../../public/images/docs/with-filter.png)
 
-Filters specify combinations of workspaces, directories, and git commits to act as entrypoints
-for execution.
+You can use it to:
 
-Multiple filters can be combined to select distinct sets of targets. Additionally, filters
-can also exclude targets. A target that matches any filter and is not explicitly excluded will
-be in the final entrypoint selection.
+- Filter by [workspace name](#filter-by-workspace-name)
```

Comment:
I _think_ technically these should all be lower cased as they are a continuation of the sentence above

Reply:
I usually don't bother with this, looks a little unnatural